test/test_various.py

```diff
@@ -2,19 +2,17 @@
 Various tests.
 """
 
-from __future__ import print_function
-
 from os.path import *
 from testsupport import *
 
 
 _files = [
```

Comment:
Yet to fix test for:
- simple/stdlib.py (just needs update of python3.x path?)
- project/foo_import.py (broken)
- project/sub1/sub11/relative_from.py (broken)